src/en/wip-systems.md

```diff
@@ -45,19 +45,33 @@ like Landscape and Nagios.
 
 ## Creating environments in a Juju System
 
+> Somewhere it should be noted that an initial juju environment must be
+> bootstrapped before 'system create-environment' can be called.
+> Later in the discussion of `juju system destroy` it becomes clear this
+> original, bootstrapped environment is the `system` and it must be
+> bootstrapped before proceeding.
+
 The `juju system create-environment` command will create a new empty
 environment that has the current user as the owner. These secondary
 environments are called 'hosted environments', as they are hosted within a
 Juju System. When creating an environment, the user will need to specify cloud
```

Comment:
Yes.  You provide the --config option to 'juju system create-environment' if the cloud provider for that type of environment requires them.  If you are creating an ec2 environment then you must provide a config. If creating a local environment then you do not.  Probably anything except local will require credentials.